jobs/build/ocp3/Jenkinsfile

```diff
@@ -647,10 +647,10 @@ node {
                         sh "tito tag --debug --accept-auto-changelog --keep-version --debug"
 
                     }
-                    if (!IS_TEST_MODE) {
```

Comment:
I commented these out because our buildvm jenkins user is no longer able to push tags to https://github.com/openshift/openshift-ansible ... I'm not suggesting that this is a good fix nor that it should be merged as-is, this was just enough to make the pipeline proceed and succeed.

But now we need to find out if the tagging is still necessary, and if so, ask the owners for writing permissions again.